egs/wsj/s5/utils/combine_lat_dirs.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+# Copyright 2016  Xiaohui Zhang  Apache 2.0.
+#                 Vijay Peddinti
+
+# This script operates on lattice directories, such as exp/tri4a_lats
+
```

Comment:
Also, the general rule is that any script that takes the 'cmd' option should be in steps/.